docs/guide-fr/installation.md

```diff
@@ -3,11 +3,11 @@ Installation
 
 ## Requirements
 
-l'extention  [MongoDB PHP Extension](http://us1.php.net/manual/en/set.mongodb.php) version 1.0.0 ou plus.
+l'extention  [MongoDB PHP Extension](https://us1.php.net/manual/en/set.mongodb.php) version 1.0.0 ou plus.
```

Comment:
Please change it to the current one - https://www.php.net/manual/en/set.mongodb.php

Reply:
done